css/templates/lune/components/_header.css

```diff
@@ -0,0 +1,68 @@
+@import "../page-vars";
+
+.ampstart-headerbar {
+  justify-content: space-between;
+
+  height: var(--layout-header-height-mobile);
+
+  /* Override ampstart default box shadow styles */
+  box-shadow: none;
+
+  /* header is fixed so should be moved onto a new GPU layer */
+  will-change: transform;
+}
+
+.ampstart-headerbar-home-link {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.ampstart-headerbar-title {
+  font-size: var(--h4);
+  font-weight: var(--font-weight-black);
```

Comment:
Is it possible to rename font-weight-black? It's not clear what this variable is for.